src/pages/Browse/index.spec.tsx

```diff
@@ -74,32 +92,55 @@ describe(__filename, () => {
     );
   });
 
-  it('calls the API to load the version info on mount', async () => {
-    const addonId = 12345;
-    const version = {
-      ...fakeVersion,
-      id: 999,
-    };
+  it('renders the content of the default file when a version has been loaded', () => {
+    const version = fakeVersion;
+
+    const store = configureStore();
+    store.dispatch(versionActions.loadVersionInfo({ version }));
+
+    const root = render({ store, versionId: String(version.id) });
+
+    expect(root.find(Highlight)).toHaveLength(1);
+    expect(root.find(Highlight)).toHaveProp('className', 'auto');
+    expect(root.find(Highlight)).toHaveProp(
+      'children',
+      ((root.instance().props as Props).file as VersionFile).content,
+    );
+  });
 
-    const mockApi = jest.spyOn(api, 'getVersion');
-    mockApi.mockReturnValue(Promise.resolve(version));
+  it('does not render the content of another file when a version has been loaded', () => {
```

Comment:
Nit: I find the description of this test to be confusing. The comment in the test helps to explain what it's testing, but I don't think this description captures it accurately.

Reply:
ah, correct. It's confusing indeed. I changed it.